config.ts

```diff
@@ -22,50 +22,81 @@ import { MKMRestaurant } from "./parsers/einpark/mkmrestaurant";
 import { ClassicRestaurantPub } from "./parsers/einpark/classicrestaurantpub";
 import { SidliskovaPivarnicka } from "./parsers/einpark/sidliskovapivarnicka";
 
+/* eslint-disable max-len */
+
+export type Location = "einpark" | "ševčenkova";
+type RestaurantMap = ReadonlyMap<Location, ReadonlyArray<{ id: number, name: string, urlFactory: (date: Date) => string, parser: IParser }>>;
+
+/**
+ * Here add new restaurant for given location
+ */
+const restaurants: RestaurantMap = new Map([

```

Comment:
extracted here the restaurant config in order to better type it + contributors have it easily accessible and separated from other configuration